tests/Backend/CommonPart1/CreateTableTest.php

```diff
@@ -250,6 +250,20 @@ public function testTableWithUnsupportedCharactersInNameShouldNotBeCreated(): vo
         }
     }
 
+    public function testBucketWithUnsupportedCharactersInNameShouldNotBeValid(): void
+    {
+        try {
+            $this->_client->createTable(
+                $this->getTestBucketId() . '.rubbish',
```

Comment:
chvilku mi trvalo, proc je takovej testcase validni, ale jasne - pred upravou by toto proslo, protoze by se udelal explode bez limitu a jako bucketId by se vzal jen $this->getTestBucketId()

Reply:
Jo, optimální je ty testy pustit nad masterem, vidět je spadnout a pak na branchí a vidět že projdou :)